notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb

```diff
@@ -0,0 +1,1056 @@
+{
```

Comment:
<p>I think it'll take at least an hour...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>True! This is copy from another notebook. Good catch.</p>